rvema/Hygieia
# Nexus IQ scan enhancements (#2391)

* Merging latest changes from master (#5)

* Fix sonar CodeQuallity collector parse error (#2028) (#2092)

* Fix TypeError: Cannot read property 'name' of undefined (#2094)

https://github.com/capitalone/Hygieia/issues/1599#issuecomment-326066852

* Chown in UI dockerfiles to change owner on nginx files (#2108)

* Chown in UI dockerfiles to change owner on nginx files

* removed useless comand

* Set DB properties to the correct defaults in all collectors (#2107)

* Mismatch between Dockerfiles and properties builder script on file names

* default DB names changed to dashboarddb

* default DB usernames changed to dashboarduser

* default DB passwords changed to dbpassword

* revert modif on score dockerfile

* Exclude merge from branch being evaluated to the branch the PR came from in case of squash merge. (#2112)

* RegressionTestResultEvaluator Bug Fix (#2116)

* Access git private repos with personal access token (#2118)

* Access git private repos with personalAccessToken

* Access git private repos with personalAccessToken

* Access git private repos with personalAccessToken

* Path correction (#2127)

* Hide label for code repo  (#2130)

* Path correction

* Hide label

* Swagger changes for Audit API; redirect score collector README (#2129)

* Access private repos with token, removed logging (#2131)

* Path correction

* Hide label

* Access privates repo with personalAccessToken. Removed logging

* Access privates repo with personalAccessToken. Removed logging

* Variables and their default are separated with : (#2122)

* Git collector continues to collect data and skip error  (#2138)

* Path correction

* Hide label

* Access privates repo with personalAccessToken. Removed logging

* Access privates repo with personalAccessToken. Removed logging

* Git collector continues to collect and skip error

* Dashboard Type Filtering over Title Filtering with Pagination (#2004)

* Changes to incorporate dashboard-type filtering along with title-based filtering

* Some minor fixes based on local testing

* Removed @Requestbody from input  (#2150)

* Path correction

* Hide label

* Access privates repo with personalAccessToken. Removed logging

* Access privates repo with personalAccessToken. Removed logging

* Git collector continues to collect and skip error

* Removed @Requestbody for input

* Modified Hudson collector - push based call (#2152)

* Fix Docker build process, score-properties-builder.sh was not valid, hygeia-jenkins-plugin failed due to mismatched dependency (#2113)

* Audit check for commits with Service account  (#2157)

* Added feature to capture LDAP_DN from Github. Added feature to detect Service Account activity in audit

* Add MergeEvent.java.

* Add COMMIT_WITH_SYSACC to direct commit

* removed junk (#2156)

* Accept personalAccessToken at repo level (#2159)

* Fix for Target and Achieved (#2165)

Testcase steps array have data in the following order Target, achieved. Currently the code is always updating Achieved property with "0" and target property with the achieved value. This fix should set target property with target value and achieved property with achieved value.

* Missing properties in properties-builder scripts See #2154 (#2168)

* UI changes for General Configuration (#2073)

* UI changes for General Congifuration

* Fixes for the pull request #2073

* Audit api - Test cases for system id check  (#2175)

* Accept personalAccessToken at repo level

* Testcase for systemId check fix

* Configuration option in admin page with collector changes & Rally collector initial commit (#2074)

* Java changes for General Configuration

* Java changes for general configuration

* Rally collector initial commit

* Fix for clearing servers in collectors collection

* Removing remaining days logic

* Fixing Remaining days in API layer rally controller

* Final fix for rallyburndown

* Update Node version to 6.9.1 (#2179)

* UI changes for rally widget (#2085)

* UI changes for rally widget

* Removed logger from the file

* UI changes for making Rally widget as standalone

* Remove company specific information in Rally widget view.js

* Sonatype UI Enhancement Code changes

* Sontaype UI enhancement code changes

* Nexus IQ scan enhancements

* Removed commented lines of code
